frontend/e2e/README.md

```
@@ -0,0 +1,348 @@
+
+# Run tests in headed mode (see browser)
+pnpm test:e2e:headed
+```
+
+## Architecture
+
+### Screenplay Pattern
+
+This test suite follows the **Screenplay pattern** to maximize maintainability. All UI selectors are centralized in dedicated files, so when the UI changes, you only need to update selectors in ONE place.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)